x-pack/plugins/security/public/views/management/edit_role/components/page_header.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiHeader,
+  EuiHeaderSection,
+  EuiHeaderBreadcrumbs
+} from '@elastic/eui';
+
+export class PageHeader extends Component {
+  render() {
+    return (
+      <EuiHeader>
+        <EuiHeaderSection>
+          <EuiHeaderBreadcrumbs breadcrumbs={this.props.breadcrumbs.map(this.buildBreadcrumb)} />
+        </EuiHeaderSection>
+      </EuiHeader>
```

Comment:
2. Can you change this to use the `EuiBreadcrumbs` like so:

```
<div>
  <EuiBreadcrumbs breadcrumbs={this.props.breadcrumbs.map(this.buildBreadcrumb)} />
  <EuiSpacer />
</div>
```

And stick in inside the `EuiPage`?

It should then look like this:

<img width="1305" alt="screen shot 2018-06-07 at 11 52 02 am" src="https://user-images.githubusercontent.com/549577/41111559-4f3620c0-6a4a-11e8-8234-983df8f25265.png">